src-java/flowhs-topology/flowhs-storm-topology/src/test/java/org/openkilda/wfm/topology/flowhs/service/FlowValidationHubServiceTest.java

```diff
@@ -110,18 +121,18 @@ public void sendInactive() {
             }
         };
         flowValidationHubService = new FlowValidationHubService(carrier, persistenceManager, flowResourcesManager,
-                MIN_BURST_SIZE_IN_KBITS, BURST_COEFFICIENT);
+                ruleManagerConfig, MIN_BURST_SIZE_IN_KBITS, BURST_COEFFICIENT);
 
         buildTransitVlanFlow("");
         flowValidationHubService.handleFlowValidationRequest(TEST_KEY, new CommandContext(),
                 new FlowValidationRequest(TEST_FLOW_ID_A));
-        for (SwitchFlowEntries switchFlowEntries : getSwitchFlowEntriesWithTransitVlan()) {
+        for (FlowDumpResponse switchFlowEntries : getFlowDumpResponseWithTransitVlan()) {
```

Comment:
just a comment, in these cases you can combine all iterable collections into a single stream:
```
Stream.of(x, y, z).flatMap(Collection::stream).forEach(...
```

Reply:
thanks, 
not sure that it is more convenient to do it via streams, cause  handleAsyncResponse throws not-runtime custom exception (UnknownKeyException extends Exception) (how to correctly handle it in the Stream.foreach() ).

But agree to collect all the collections in the only one and get through the for each loop only ones for entire collection.